CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.2] - 2019-06-12
```

Comment:
What should be the version change of this Docker image ? Should it be a minor version ?

We have a major version change in Shunting Yard. I think in Apiary-metastore-docker, we changed the minor version when we upgraded apiary-extensions to a major version. I guess, I can do the same ?

Reply:
https://github.com/ExpediaGroup/apiary-metastore-docker/blob/master/CHANGELOG.md

Reply:
https://github.com/ExpediaGroup/apiary-metastore-docker/blob/master/CHANGELOG.md

Reply:
Yeah, that thought also crossed my mind, I guess at this level it would depend on whether the change to that would affect the user of the Docker image. If they don't notice then I think a minor or patch version update is fine (like you've done).